Cargo.toml

```diff
@@ -13,7 +13,7 @@ crate-type = ["cdylib"]
 [dependencies]
 skyline = { git = "https://github.com/ultimate-research/skyline-rs.git" }
 skyline_smash = { git = "https://github.com/ultimate-research/skyline-smash.git" }
-smush_discord_shared = { git = "https://github.com/jam1garner/smush_info_shared.git" }
+smush_info_shared = { git = "https://github.com/leeevye/smush_info_shared.git", branch = "feature(const_fn_union)" }
```

Comment:
minor issue this seems like it needs to be removed

Reply:
This is an interesting one, latest cargo won't compile smush_discord while smush_info_shared includes a line of code that cargo skyline won't compile without. I created a new branch to include `#![feature(const_fn_union)]` and I use that branch here.

Reply:
could this maybe be a cargo feature instead? then use something like:
```rust
#![cfg_attr(feature = "const_fn_union", feature(const_fn_union))]
```
and in Cargo.toml:
```toml
[features]
const_fn_union = []
```